ZuluContent/Spells/Seventh/Polymorph.cs

```diff
@@ -141,12 +141,16 @@ public async Task CastAsync()
                 return;
 
             var hueMod = bodyId == SeaSerpent.BodyId ? 233 : 0;
+            var statMod = SpellHelper.GetModAmount(Caster, Caster, StatType.All);
+            var arMod = statMod / 3;
             
             Caster.TryAddBuff(new Polymorph
             {
+                Description = $"<br>Str: +{statMod}<br>Dex: +{statMod}<br>Int: +{statMod}<br>Armor: +{arMod}",
                 Duration = SpellHelper.GetDuration(Caster, Caster),
                 BodyMods = (bodyId, hueMod),
-                Value = SpellHelper.GetModAmount(Caster, Caster, StatType.All)
+                StatMods = (StrMod: statMod, DexMod: statMod, IntMod: statMod),
+                ArmorMod = arMod,
```

Comment:
Mmm I guess I would do something similar here for Earth's Blessing just to provide armor in 1 buff

Reply:
Yeah probably preferable to have them as one buff for UX sake